pincer/utils/api_object.py

```diff
@@ -210,7 +210,11 @@ def __post_init__(self):
                     self.__attr_convert(attr_item, classes[0])
                     for attr_item in attr_gotten
                 ]
-
+            elif tp == dict and attr_gotten and (classes := get_args(types[0])):
```

Comment:
I **think** the parentheses are not needed here
```suggestion
            elif tp is dict and attr_gotten and classes := get_args(types[0]):
```

Reply:
They are, I checked no parentheses.

Reply:
Okay my bad then :( feel free to resolve this